README.md

```diff
@@ -91,7 +91,9 @@ Install all required packages and gems.
 
 ```shell
 pkg install cmake pkgconf
+pkg install mysql-connector-c-6.1.11
```

Comment:
Since this only applies for users of the `MySQL` source, it seems like this would be a better fit for `docs/Sources.md` under the `Source: MySQL` section?